grc/core/params/param.py

```diff
@@ -22,6 +23,8 @@
 @setup_names
 class Param(Element):
 
+    EvaluationType = Union[None, str, complex, float, int, bool, List[str], List[complex], List[float], List[int]]
```

Comment:
If something can be None, would it make more sense for the type to be `Union[all possible types except None]`, then later use `Optional[EvaluationType]`? I'm not sure which is most "pythonic"

Reply:
Yeah, that would be equivalent. I opted for explicitly listing `None`, because that's one possible evaluation. (Kind of ambiguous what `None` means here)